src/serviceWorker/index.ts

```diff
@@ -33,5 +37,5 @@ precacheAndRoute(precacheEntries) // precache cache
 // We only use the precache and runtime caches, so we delete the rest to avoid taking space.
 // Wait to do so until 'activate' in case activation fails.
 self.addEventListener('activate', () =>
-  deleteUnusedCaches(self.caches, { usedCaches: [cacheNames.runtime, cacheNames.precache] })
+  deleteUnusedCaches(self.caches, { usedCaches: [cacheNames.precache, onDemandCacheName] })
```

Comment:
should this be `preCache`?

Reply:
no, this comes from `workbox-core`, and is the cache automatically used by `precacheAndRoute`.
https://developer.chrome.com/docs/workbox/reference/workbox-core/#property-cacheNames